en/docs/administer/monitoring-logs.md

```diff
@@ -152,70 +159,81 @@ Following are the log levels that can be configured:
 
-    -   `            log4j.appender.CARBON_LOGFILE.MaxBackupIndex=20           `
+You can limit the size of the
+`         <IS_HOME>/repository/logs/wso2carbon.log        `
+file by configuring Rollover based on log file size by following the steps given below.
+
+1.  Remove the
+       ``` toml
+       appender.CARBON_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+       appender.CARBON_LOGFILE.policies.time.interval = 1
+       ```
+    add the 
+       ``` toml
+       appender.CARBON_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+       appender.CARBON_LOGFILE.policies.size.size=10MB
+       ```
+    in the
 
+    `           <PRODUCT_HOME>/repository/conf/          `
+    `           log4j2.properties          ` file.
+    
     If the size of the log file is exceeding the value defined in the
-    `            MaxFileSize           ` property, the content is copied
+    `            appender.CARBON_LOGFILE.policies.size.size           ` property, the content is copied
     to a backup file and the logs are continued to be added to a new
-    empty log file. The `            MaxBackupIndex           ` property
-    makes the Log4j maintain a maximum number of backup files for the
-    logs.
+    empty log file.
+    
+
+2.  Following property under `CARBON_LOGFILE` appender is to limit the number of backup files.
+    You can change it as per your requirement by changing its value.
+
+    -   `            appender.CARBON_LOGFILE.strategy.max           `
 
 #### Limiting the size of audit log files
 
 In WSO2 servers, audit logs are enabled by default. We can limit the
 audit log files with the following configuration:
 
-1.  Change the
-    `          log4j.appender.AUDIT_LOGFILE=org.wso2.carbon.logging.appenders.CarbonDailyRollingFileAppender         `
-    appender in the
-    `          <IS_HOME>/repository/conf/log4j.properties         `
-    file as follows:
-    `          log4j.appender.AUDIT_LOGFILE=org.apache.log4j.RollingFileAppender         `
-2.  Add the following two properties under
-    `          RollingFileAppender         `:
-    -   `            log4j.appender.AUDIT_LOGFILE.MaxFileSize=10MB           `
-    -   `            log4j.appender.AUDIT_LOGFILE.MaxBackupIndex=20           `
+1.  Remove the
+       ``` toml
+       appender.AUDIT_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+       appender.AUDIT_LOGFILE.policies.time.interval = 1
+       ```
+    from the `AUDIT_LOGFILE` appender in the
+    `          <IS_HOME>/repository/conf/log4j2.properties         `
+    add the the following to the `AUDIT_LOGFILE` appender :
+       ``` toml
+    appender.AUDIT_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+    appender.AUDIT_LOGFILE.policies.size.size=10MB
+       ```
+2.  Following property under `AUDIT_LOGFILE` appender is to limit the number of backup files.
+    You can change it as per your requirement by changing its value.
```

Comment:
please change to "**The** following property  under the `AUDIT_LOGFILE` appender is **used** to...."

Reply:
done

Reply:
"Following" should start with simple "f"